migtests/scripts/run-test.sh

```diff
@@ -173,9 +173,10 @@ main() {
 	run_ysql ${TARGET_DB_NAME} "\dft" 
 
 	step "Run validations."
-	if [ -x "${TEST_DIR}/validate" ]
-	then
-		 "${TEST_DIR}/validate"
+	if [[ "${EXPORT_TABLE_LIST}" != "" && -x "${TEST_DIR}/validate-with-table-list" ]]; then
+		"${TEST_DIR}/validate-with-table-list"
+	elif [[ -x "${TEST_DIR}/validate" ]]; then
+		"${TEST_DIR}/validate"
```

Comment:
@priyanshi-yb @makalaaneesh 
Just thinking if special handling like this for table-list case might complicate the automation framework.
From what I understood we are handling multiple expected files differently here. like export data status and import data status. 

Instead can we just simply add a new test here - `pg/sequence-with-tablelist`?

Reply:
Somehow, this seems easier to me to add the table-list test this way. We can take it up later when we plan to include live tests as well for these cases: the full list and subset of the table list.

Reply:
yeah np, we can take this later.
To me this approach seems more useful when we plan to add table list related test/validation in a lot of tests. Since it increases the complexity of the framework.

Otherwise for a single test, we can separate this, even if there is some duplication or repeatition.